tests/tck/features/match/Scan.feature

```diff
@@ -78,6 +78,67 @@ Feature: Match seek by scan
       | Name   |
       | "Mary" |
 
+  Scenario: query vertices by scan with skip limit
+    When executing query:
+      """
+      MATCH (v)
+      RETURN v.person.name AS name
+      SKIP 10 LIMIT 4
+      """
+    Then the result should be, in any order:
+      | name     |
+      | /[\w_]+/ |
+      | /[\w_]+/ |
+      | /[\w_]+/ |
+      | /[\w_]+/ |
```

Comment:
better to sort the results and compare the real values

Reply:
```
      MATCH (v)
      RETURN v.person.name AS name
      ORDER BY name
      SKIP 10 LIMIT 4
```

Reply:
Scan don't support push down sort.